configure.ac

```diff
@@ -1866,6 +1866,20 @@ if test "$OPENSSL_ENABLED" = "1"; then
    ])
 fi
 
+dnl ---
+dnl Whether the OpenSSL configuration will be loaded automatically
+dnl ---
+if test X"$OPENSSL_ENABLED" = X"1"; then
+AC_ARG_ENABLE(ssl-auto-load-config,
+AC_HELP_STRING([--enable-ssl-auto-load-config],[Enable automatic loading of OpenSSL configuration])
+AC_HELP_STRING([--disable-ssl-auto-load-config],[Disable automatic loading of OpenSSL configuration]),
```

Comment:
Since it's OpenSSL specific, shouldn't the switch be `--disable-openssl-auto-load-config` to avoid confusion with a generic flag supported by any vtls implementation?

Reply:
@pwaehnert would you mind renaming this option as per @danielgustafsson's suggestion? Since this is very much an openssl-specific option it makes perfect sense.